src/server/rdb_load.cc

```diff
@@ -69,6 +69,76 @@ inline void YieldIfNeeded(size_t i) {
   }
 }
 
+// taken from zset.c
+unsigned char* zzlInsertAt(unsigned char* zl, unsigned char* eptr, sds ele, double score) {
```

Comment:
I think it'd be cleaner to have this and the below functions in the zset-specific files. But that might require a larger refactor than what you had hoped to do, so.. your call.

Reply:
it's a remporary code anyway. I am planning to introduce our own zset and this is only a preparation.